web/src/lib/components/admin-page/settings/setting-input-field.svelte

```diff
@@ -39,8 +40,16 @@
 			</div>
 		{/if}
 	</div>
+
+	{#if desc}
+		<p class="immich-form-label text-xs pb-2" id="{label}-desc">
+			{desc}
+		</p>
+	{/if}
+
 	<input
-		class="immich-form-input w-full"
+		class="immich-form-input pb-2 w-full"
+		aria-describedby={desc ? `${label}-desc` : undefined}
```

Comment:
Might as well add `aria-labelledby="{label}-label` too (need to to add `id="{label}-label"` to the label)